src/main/java/world/bentobox/bentobox/api/commands/island/team/IslandTeamKickCommand.java

```diff
@@ -76,13 +77,20 @@ private void kick(User user, UUID targetUUID) {
             if (target.isOnline()) {
                 target.getPlayer().getEnderChest().clear();
             }
-            // FIXME need some special handling here if the target's offline.
+            else {
+                this.getAddon().getPlayers().getPlayer(targetUUID).addQuarantinedWorld(getWorld());
```

Comment:
```suggestion
                getPlayers().getPlayer(targetUUID).addQuarantinedWorld(getWorld());
```
Use that, it's cleaner.

Reply:
Use that on all lines that do a similar call ^^